repl/src/main/scala/ammonite/repl/interp/Interpreter.scala

```diff
@@ -135,8 +135,16 @@ class Interpreter(shellPrompt0: Ref[String],
     def search(target: scala.reflect.runtime.universe.Type) = Interpreter.this.compiler.search(target)
     def compiler = Interpreter.this.compiler.compiler
     def newCompiler() = init()
-    def history = history0.toVector.dropRight(1)
+    def history = history0
     def show[T](a: T, lines: Int = 0) = ammonite.pprint.Show(a, lines)
+
+    implicit val historyPPrinter: pprint.PPrinter[History] = 
```

Comment:
Now I see that this instance doesn't belong in ReplAPI, it should be in the History companion.